components/progressbar/metadata/meter.yml

```diff
@@ -1,34 +1,71 @@
 name: Meter
-description: Meter is implemented using the [Progress bar](progressbar.html) component. Refer to the [Progress bar API docs](progressbar.html) for API details.
+description: |
+  Meter is implemented using the [Progress bar](progressbar.html) component. Refer to the [Progress bar API docs](progressbar.html) for API details.
+  Meter should be only used in size `S` and `L` and could use the meter t-shirt sized classnames. [The Meter table of options](https://spectrum.adobe.com/page/meter/#Table-of-options) provides further details on which variants are available.
+
+  | Previous size classname         | New size classname                |
+  | ------------------------------- | --------------------------------- |
+  | `.spectrum-ProgressBar--sizeS`  | `.spectrum-Meter--sizeS`    |
+  | `.spectrum-ProgressBar--sizeL`  | `.spectrum-Meter--sizeL`    |
 SpectrumSiteSlug: https://spectrum.adobe.com/page/meter/
+
 examples:
+  - id: meter-sizes
+    name: Sizing
+    markup: |
+      <div style="margin: 20px 0;">
+        <div class="spectrum-ProgressBar spectrum-Meter--sizeS is-positive" value="50" role="progressbar" aria-valuenow="50" aria-valuemin="0" aria-valuemax="100" style="width: 200px;">
```

Comment:
Is `.spectrum-Meter--sizeS` supposed to replace `.spectrum-ProgressBar--sizeS`? I see they are both used together in these examples.